portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -240,12 +240,37 @@ definition {
 	}
 
 	@description = "Verify that the Field Title can be edited"
-	@ignore = "true"
 	@priority = "5"
 	test TitleCanBeEdited {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131226 TitleCanBeEdited pending implementation
+	ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+	FormsAdminNavigator.gotoAddForm();
+
+	Form.editName(formName = "Form 1");
+
+	Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
```

Comment:
Hi @piresdm,

Soon, another pull request will change the macro `gotoAddField` and it'll consider the `fieldPositionNumber` starting from `0` instead of `1`, in light of this can you please set the `fieldPositionNumber` to `0`?

From now on, this behavior of starting the count using `0` instead of `1` will be the default behavior. Please consider this information for your next pulls. Sorry for any inconvenience it may cause to you. :slightly_smiling_face:

Reply:
Done.
Thanks.